### PR TITLE
Do not try to fulltext-index large files.

### PR DESCRIPTION
**Pull request by aduffeck**
Both tika and ocis do not handle large files/archives very well.

### CODE REVIEW

**Comment by ownclouders** [comment]
E2E tests failed: https://drone.owncloud.com/owncloud/ocis/22735/69/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/ocis/22735/tracing/public-link-for-space-alice-2023-5-26-08-45-15.zip 


</p></details>

File: services/search/README.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ As soon as Tika is installed and accessible, the search service must be configur
 
 When the search service can reach Tika, it begins to read out the content on demand. Note that files must be downloaded during the process, which can lead to delays with larger documents.
 
+Content extraction and handling the extracted content can be very resource intensive. Therefor content extraction is limited to files with a certain file size. The default limit is 20MB and can be configured using the `SEARCH_CONTENT_EXTRACTION_SIZE_LIMIT` variable.
```
Comment (mmattel) [inline services/search/README.md:55]:
```suggestion
Content extraction and handling the extracted content can be very resource intensive. Content extraction is therefore limited to files with a certain file size. The default limit is 20MB and can be configured using the `SEARCH_CONTENT_EXTRACTION_SIZE_LIMIT` variable.
```
Therefore is an adverb that means “as a consequence,” “as a result,” or “hence.”
Therefor is an adverb that means “for that,” or “for it.”

File: services/search/pkg/config/config.go [hunk 2]
```diff
@@ -18,11 +18,12 @@ type Config struct {
...
-	GRPCClientTLS *shared.GRPCClientTLS `yaml:"grpc_client_tls"`
-	Events        Events                `yaml:"events"`
-	Engine        Engine                `yaml:"engine"`
-	Extractor     Extractor             `yaml:"extractor"`
+	Reva                       *shared.Reva          `yaml:"reva"`
+	GRPCClientTLS              *shared.GRPCClientTLS `yaml:"grpc_client_tls"`
+	Events                     Events                `yaml:"events"`
+	Engine                     Engine                `yaml:"engine"`
+	Extractor                  Extractor             `yaml:"extractor"`
+	ContentExtractionSizeLimit uint64                `yaml:"content_extraction_size_limit" env:"SEARCH_CONTENT_EXTRACTION_SIZE_LIMIT" desc:"Maximum file size in bytes that is allowed for the content extraction."`
```
Comment (mmattel) [inline services/search/pkg/config/config.go:26]:
```suggestion
	ContentExtractionSizeLimit uint64                `yaml:"content_extraction_size_limit" env:"SEARCH_CONTENT_EXTRACTION_SIZE_LIMIT" desc:"Maximum file size in bytes that is allowed for content extraction."`
```

**Review by mmattel — APPROVED** [review]
LGTM from a docs pov

**Review by butonic — APPROVED** [review]

**Pull request merged**